### PR TITLE
Adde delete/update in gsheet and trigger mail

### PR DESCRIPTION
**Pull request by athiruma**
1. Added append/delete to google spreadsheet
2. Added mail notification to ec2_idle & ec2_stop after stop/delete.

### CODE REVIEW

File: cloud_governance/policy/ec2_idle.py [hunk 1]
```diff
@@ -147,17 +147,18 @@ def __trigger_mail(self, user: str, resource_id: str, metrics: list):
...
-        subject = f'Instance- {resource_id} idle more than {self.INSTANCE_IDLE_MAIL_NOTIFICATION_DAYS} days'
+        subject = f'cloud-governance alert: Stopped ec2-idle more than 4 days'
         body = f"""
 Hi,
 
-Instance: {resource_id} is idle since {self.INSTANCE_IDLE_MAIL_NOTIFICATION_DAYS}.  
-After {self.STOP_INSTANCE_IDLE_DAYS} days of instance is idle we stop that instance. If you don't want to stop this instance add Policy=Not_Delete tag to your instance. 
+Instance: {resource_id} is idle more than {days} days.
+This instance will be stopped. 
+If you you want that cloud-governance will not stop it add Policy=Not_Delete tag to your instance.
```
Comment (ebattat) [inline cloud_governance/policy/ec2_idle.py:156]: If you want that cloud-governance will not stop it add Policy=Not_Delete tag to your instance.

Comment (athiruma) [reply cloud_governance/policy/ec2_idle.py:156]: Both are same

File: .github/workflows/Build.yml [hunk 2]
```diff
@@ -215,14 +215,17 @@ jobs:
...
   gitleaks:
     name: gitleaks
     needs: [ test, pypi_upload, pypi_validate, quay_upload, bump_version ]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: run gitleaks
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.SECRET_ACCESS_KEY }}
```
Comment (ebattat) [inline .github/workflows/Build.yml:228]: Are u sure we need AWS key for it ?

Comment (athiruma) [reply .github/workflows/Build.yml:228]: not sure, but we are uploading output to s3. So we need

File: cloud_governance/policy/ec2_idle.py [hunk 3]
```diff
@@ -147,17 +147,18 @@ def __trigger_mail(self, user: str, resource_id: str, metrics: list):
...
-{", ".join(metrics)}
 
-Thanks
-Thirumalesh""".strip()
+Best Regards,
+Thirumalesh
+Cloud-governance Team""".strip()
         self._mail.send_mail(receivers_list=receivers_list, body=body, subject=subject)
+
+
```
Comment (ebattat) [inline cloud_governance/policy/ec2_idle.py:164]: extra enter

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**